components/ConfirmPopup.vue

```diff
@@ -17,7 +17,7 @@
         id="confirmation"
         v-model="confirmation_typed"
         type="text"
-        placeholder="Type your username to continue"
+        placeholder="Type the input needed to continue"
```

Comment:
seems strangely worded, but that works I suppose.

Reply:
If you have a better suggestion, please tell it, because I was stuck for a good minute trying to find a better wording

Reply:
I thought about it for a bit but couldn't come up with anything better aside from maybe exposing a prop for the component to have it defined on a case by case basis.

Reply:
I prefer to avoid adding another prop to the mix, it will only add more complexity to the component